hubstorage/batchuploader.py

```diff
@@ -3,12 +3,14 @@
 import random
 import logging
 import warnings
+import six
```

Comment:
Thanks for catching this.

Added `six` and rebased against `master`. #64 allowed to catch 2 more issues.